azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -139,9 +139,9 @@ def handle_pipeline_event(self, event):
         try:
             self._handle_pipeline_event(event)
         except Exception as e:
-            # Do not use exc_info parameter on logger.* calls.  This casuses pytest to save the traceback which saves stack frames which shows up as a leak
+            # Do not use exc_info parameter on logger.* calls.  This causes pytest to save the traceback which saves stack frames which shows up as a leak
             logger.error(msg="Unexpected error in {}._handle_pipeline_event() call".format(self))
-            handle_exceptions.handle_background_exception(e)
```

Comment:
>handle_exceptions.handle_background_exception(e) [](http://example.com/codeflow?start=12&length=48)

Can you remove handle_background_exception from the handle_exceptions module and stop importing it?